content/en/resources/_index.md

```diff
@@ -1,8 +1,10 @@
 ---
-title:  "Resources"
+title: "Resources"
 description: "Kubernetes Community Resources"
 type: docs
 menu:
   main:
     weight: 50
 ---
+
+##### [Kubernetes v1.31: Elli](https://kubernetes.io/blog/2024/08/13/kubernetes-v1-31-release)
```

Comment:
Hm, I have no idea how to format that link properly into the section index :thinking: